R/ui.R

```diff
@@ -180,8 +180,11 @@ show_block_panel <- function(id, add_panel = TRUE, session = get_session()) {
 
   ns <- session$ns
 
-  if (add_panel) {
+  if (add_panel && !block_panel_id(id) %in% list_block_panels(session)) {
```

Comment:
@DivadNojnarg this is an example of an issue I get with the `input`-based functions like `get_panels_ids ()`: I wanted to create a "smart" `show_block_panel()` function that auto-adds a panel if it does not already exists. But with the current form of `get_panels_ids ()` I can never be sure of the result I get. It might be that the panel actually is there, but just not visible yet in `input`. This lead me to https://github.com/cynkra/dockViewR/issues/49#issuecomment-3357133722.

Reply:
Given that what I suggest in https://github.com/cynkra/dockViewR/issues/49#issuecomment-3357133722 is not possible, we'll have to come up with something else.